src/provider/cloud-element-templates/cmd/ChangeElementTemplateHandler.js

```diff
@@ -452,12 +457,36 @@ export default class ChangeElementTemplateHandler {
       });
     }
   }
+
+  /**
+   * Replaces the element with the specified elementType
+   *
+   * @param {djs.model.Base} element
+   * @param {Object} newTemplate
+   */
+  _updateTaskType(element, newTemplate) {
+
+    // determine new task type
+    const newType = newTemplate.elementType;
+
+    if (!newType) {
+      return element;
+    }
+
+    // don't replace Task that is already the correct type
+    if (is(element, newType.value)) {
```

Comment:
So we don't allow morphing ("back") into parent types? E.g.

* element has type `serviceTask`
* `template.elementType` is `task`
* do not morph element to `task`

Which makes sense due to our story I guess, just want to make sure.

Reply:
Let's be explicit here: If I want to replace a SubProcess with an AdHocSubprocess, this would break as well.

We don't support SubProcesses for this feature yet, but we will be on the safe side if we want to include them in the future

Reply:
So: don't use the meta-model and allow morphing to general Task